src/observable/FromEventObservable.ts

```diff
@@ -78,28 +84,31 @@ export class FromEventObservable<T, R> extends Observable<T> {
    */
   static create<T>(target: EventTargetLike,
                    eventName: string,
-                   selector?: (...args: Array<any>) => T): Observable<T> {
-    return new FromEventObservable(target, eventName, selector);
+                   selector?: (...args: Array<any>) => T,
+                   options?: EventListenerOptions): Observable<T> {
+    return new FromEventObservable(target, eventName, selector, options);
```

Comment:
maybe this can be polymorphic to support pass options without selector
such as below?
```js
fromEvent(target,event,options);
fromEvent(target,event,selector, options);
```